.vscode/extensions.json

```diff
@@ -1,7 +1,28 @@
 {
-	"recommendations": [
-		"dbaeumer.vscode-eslint",
-        "eamodio.gitlens",
-        "firsttris.vscode-jest-runner"
-	]
-}
\ No newline at end of file
+  "recommendations": [
+    "bierner.github-markdown-preview",
+    "circleci.circleci",
+    "dbaeumer.vscode-eslint",
+    "eamodio.gitlens",
+    "esbenp.prettier-vscode",
+    "EditorConfig.EditorConfig",
+    "firsttris.vscode-jest-runner",
+    "GitHub.remotehub",
+    "github.vscode-pull-request-github",
+    "gruntfuggly.todo-tree",
+    "mechatroner.rainbow-csv",
+    "ms-azuretools.vscode-docker",
+    "ms-vscode.remote-repositories",
+    "ms-vscode.vscode-js-profile-flame",
+    "ms-vsliveshare.vsliveshare",
+    "oderwat.indent-rainbow",
+    "quicktype.quicktype",
+    "rangav.vscode-thunder-client",
+    "streetsidesoftware.code-spell-checker",
+    "timonwong.shellcheck",
+    "tlahmann.alex-linter",
+    "tyriar.sort-lines",
+    "unifiedjs.vscode-mdx",
+    "usernamehw.errorlens"
+  ]
```

Comment:
Some stuff in here I'll have to explore more as I'm unfamiliar but is all seems great

 a few others i like and use
 
<Author> - <extension name>

avrammavridis - React Documentation
dsznajder - ES7+ React/Redux/React-Native snippets
cardinal90 - Multiple cursor casepreserve

Reply:
Oh also Simon Siefke - Svg preview

Reply:
Oh also also ChkrounAnas - Turbo Console Log 
Tis nice

Reply:
I'm adding all these; they seem to be quite helpful, thanks! With respect to the SVG preview option, [this SVG-related extension](https://marketplace.visualstudio.com/items?itemName=jock.svg) seems to be more popular. Anything the SVG Preview extension does that this one can't?